src/routes.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 
-import { LandingPage, SecondaryPage } from './components';
+import { LandingPage, SecondaryPage, BusinessSignUp, VolunteerSignUp } from './components';
 
 export default function Routes() {
   return (
     <Switch>
+      <Route exact path='/' component={LandingPage} />
```

Comment:
When using a Switch to render routes, we put the base route at the bottom and have no need for the exact property. So what you can do is put the index path `<Route exact path='/' component={LandingPage} />` at the bottom of the Route list and remove the exact property.